source/retryable-reads/tests/README.rst

```diff
@@ -232,6 +232,56 @@ This test requires MongoDB 4.2.9+ for ``blockConnection`` support in the failpoi
 
 9. Disable the failpoint.
 
+Retrying Reads in a Sharded Cluster
+===================================
+
+These tests will be used to ensure drivers properly retry reads on a different
+mongos.
+
+Retryable Reads Are Retried on a Different Mongos if One Available
+------------------------------------------------------------------
+
+This test MUST be executed against a sharded cluster that has at least two
+mongos instances.
+
+1. Ensure that a test is run against a sharded cluster that has at least two
+   mongos-es. If there are more than two mongos-es in the cluster, pick two to
+   test against.
+
+2. Create a client per mongos using the direct connection, and configure fail
+   points on each of the picked mongos-es, so that each mongos raises
+   a retryable error once.
+
+3. Create a client with ``retryReads=true`` that connects to the cluster,
+   providing the two selected mongos-es as seeds.
+
+4. Enable command monitoring, and execute a read command that is
+   supposed to fail on both mongos-es.
+
+5. Asserts that there were failed command events from each mongos.
+
+6. Disable the fail points.
+
+
+Retryable Reads Are Retried on the Same Mongos if No Other Available
+--------------------------------------------------------------------
+
+1. Ensure that a test is run against a sharded cluster. If there are multiple
+   mongos-es in the cluster, pick one to test against.
+
+2. Create a client that connects to the mongos using the direct connection,
+   and configure a fail point so that the mongos raises a retryable error once.
```

Comment:
I suggest noting not to set a failpoint error that is transient, according to the [specifications](https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#why-are-read-operations-only-retried-once-by-default). See other comment for more details.